control_software/src/paper_gpu/utils.py

```diff
@@ -13,3 +14,20 @@ def run_on_hosts(hosts, cmd, user=None, wait=True):
             pn.wait()
     else:
         return p
+
+def get_current_jd():
+    """
+    Get the current Julian date (JD) from Unix time.
+
+    Paramaters
+    ----------
+    None
+
+    Returns
+    -------
+    float : the current Julian date
+    """
+    unix_time = time.time()
+
+    # JD 2440587.5 == Jan 1, 1970
+    return (unix_time / 86400.0) + 2440587.5
```

Comment:
Is there a less hardcoded way to do this? I always want to avoid magic numbers in code.

Reply:
ChatGPT suggests:
```from astropy.time import Time
import time

# Get current Unix time
current_unix_time = time.time()

# Convert Unix time to Julian Day
time_obj = Time(current_unix_time, format='unix')
julian_day = time_obj.jd

print("Current Julian Day:", julian_day)```
which seems to work

Reply:
Well if ChatGPT says it, then it must be the best way to do it 😉 

I was trying to avoid an explicit astropy dependence, especially because sometimes time calculations cause errors if up-to-date leap seconds aren't available (though this seems to have improved somewhat in recent versions). I'll make a change to astropy.